torch/csrc/jit/passes/tensorexpr_fuser.cpp

```diff
@@ -1236,7 +1249,8 @@ void FuseTensorExprs(
 }
 
 Operation createTensorExprOp(const Node* node) {
-  if (!tensorExprDynamicShapeFusionEnabled()) {
+  bool dynamic_shape_fusion_node = node->hasAttribute(attr::striding_inputs_desc);
```

Comment:
Is every `TensorExprGroup` node with dynamic shapes guaranteed to have `attr::striding_inputs_desc`? What if there are no dynamic strides?

Reply:
yes always guaranteed. If there are no dynamic strides then there will be `attr::striding_inputs_desc` with TENSOR_CONT description